recipes/libbasisu/all/test_package/CMakeLists.txt

```diff
@@ -6,5 +6,7 @@ conan_basic_setup(TARGETS)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 
+target_compile_definitions(${PROJECT_NAME} PRIVATE BASISU_NO_ITERATOR_DEBUG_LEVEL=1)
```

Comment:
If it's needed for the test package.. it should be in the recipe

Reply:
Because this "magic with iterators" is located in the header `transcoder/basisu.h`, so any including this header has to define this preprocessor directive, if don't want to use custom iterator level, or it gets error LNK2038 (value '2' doesn't match value '1' in test_package.obj). But as I found it could be solved using define in cpp_info structure, filled in method package_info and I already rebase last commit to this solution.